src/bin/cargo-flamegraph.rs

```diff
@@ -67,6 +67,10 @@ struct Opt {
     #[structopt(short = "f", long = "features")]
     features: Option<String>,
 
+    /// Disable default features
+    #[structopt(short = "n", long = "no-default-features")]
+    no_def_features: bool,
```

Comment:
Let's not have a short option for this (Cargo doesn't either), and let's expand the field to the full name.

Reply:
Done